CentralComputing/Utils.cpp

```diff
@@ -40,20 +41,20 @@ void Utils::print(LogLevel level, const char * format, ...){
       }
 }
 
-int Utils::toogleGPIOUp(int GPIONumber) {
-    std::string start = "/sys/class/gpio/gpio"
+int Utils::set_GPIO_high(int GPIONumber) {
+    std::string start = "/sys/class/gpio/gpio";
     std::string integer = std::to_string(GPIONumber);
-    std::string end = "/value"
+    std::string end = "/value";
     std::string path = start + integer + end;
     std::ofstream out(path);
```

Comment:
This might work as it is, but please add the truncate option. Since normally that is how these files are treated, the `echo 1 > file` indicates a truncate, not an append. 
http://www.cplusplus.com/reference/fstream/ofstream/ofstream/